src/Algorithms/compute_Fext.jl

```diff
@@ -0,0 +1,11 @@
+function compute_Fext!(alg::ArcLength, varFext, currλ, time, Fextk, args...)
```

Comment:
El bang (`!`) se utiliza tipicamente con funciones que cambian un argumento en el sitio (in-place), pero no es el caso. 

De hecho, esto genera un vector cada vez, pero no interesa la historia ya que `ModeSol` guarda solo un `Fextk::Vector{Float64}`. Por tanto, seria mejor que actualice el campo de `ModelSol` directamente (lo cual no implica que `ModelSol` sea mutable, ya que `Vector` lo es).

Reply:
En ppio basta con hacer `ModelSol.Fext .= ...` y en cada operacion, o `@.  (...)` adelante de la expresion.

Reply:
Gracias a ambos!